benchmark-service/benchmark/static/js/autocomplete.js

```diff
@@ -0,0 +1,95 @@
+
+    new autoComplete({
```

Comment:
What I was thinking for the deduplication is whether we could create a function that took some parameters that returned a new AutoComplete object/function. I'm not sure if that's possible.

@danking Can you please advise?